jest.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  preset: "ts-jest",
+  testEnvironment: "node",
+  transform: {
+    "^.+\\.tsx?$": "ts-jest",
+    "^.+\\.jsx?$": "babel-jest"
```

Comment:
Do we need to specify this? I thought jest uses the babel transform automatically (and babel supports typescript).

In Vega-Lite, we almost don't need to override ay configs: https://github.com/vega/vega-lite/blob/master/package.json#L141